From d652c509b4701f9fffcc935ddf04a6164d2b801a Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Tue, 15 Sep 2020 17:48:17 +0200
Subject: [PATCH 7/7] Register Ubuntu.Test plugin as module

Fixes the following error during make install:

file:///typelist.qml:15:1: plugin cannot be loaded for module "Ubuntu.Test": Cannot protect module Ubuntu.Test 0 as it was never registered
---
 src/imports/Test/plugin/testplugin.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/imports/Test/plugin/testplugin.cpp b/src/imports/Test/plugin/testplugin.cpp
index c7a63c512..02a5c266a 100644
--- a/src/imports/Test/plugin/testplugin.cpp
+++ b/src/imports/Test/plugin/testplugin.cpp
@@ -33,6 +33,7 @@ static QObject *registerExtras(QQmlEngine *engine, QJSEngine *scriptEngine)
 
 void TestPlugin::registerTypes(const char *uri)
 {
+    qmlRegisterModule(uri, 0, 1);
     qmlRegisterSingletonType<UCTestExtras>(uri, 1, 0, "TestExtras", registerExtras);
     qmlRegisterSingletonType<MouseTouchAdaptor>(uri, 1, 0, "MouseTouchAdaptor", MouseTouchAdaptor::registerQmlSingleton);
 }
-- 
2.28.0

